src/components/tx/SignOrExecuteForm/ExecuteForm.tsx

```diff
@@ -137,17 +137,21 @@ const ExecuteForm = ({
           <ErrorMessage>
             Cannot execute a transaction from the Safe Account itself, please connect a different account.
           </ErrorMessage>
-        ) : executionValidationError || gasLimitError ? (
-          <ErrorMessage error={executionValidationError || gasLimitError}>
-            This transaction will most likely fail.
-            {` To save gas costs, ${isCreation ? 'avoid creating' : 'reject'} this transaction.`}
-          </ErrorMessage>
         ) : (
-          submitError && (
-            <ErrorMessage error={submitError}>Error submitting the transaction. Please try again.</ErrorMessage>
+          (executionValidationError || gasLimitError) && (
+            <ErrorMessage error={executionValidationError || gasLimitError}>
+              This transaction will most likely fail.
+              {` To save gas costs, ${isCreation ? 'avoid creating' : 'reject'} this transaction.`}
+            </ErrorMessage>
           )
         )}
 
+        {submitError && (
+          <Box mt={1}>
+            <ErrorMessage error={submitError}>Error submitting the transaction. Please try again.</ErrorMessage>
+          </Box>
+        )}
```

Comment:
This error must be shown separate of validation errors because the user can still submit a tx and this error has been only logged in the console.